### PR TITLE
production template: update traefik command

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

**Review by tranchitella — APPROVED** [review]

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

File: production/config/prod.yml.template [hunk 1]
```diff
@@ -80,6 +80,7 @@ services:
             - --entryPoints.https.transport.respondingTimeouts.writeTimeout=7200
```
Comment (alfrunes) [inline production/config/prod.yml.template:80]: Can you also remove the following unused arguments as well?
```yaml
             --providers.file.filename=/config/tls.toml # File no longer exists
             --providers.docker=true
             --providers.docker.exposedbydefault=false
```

Comment (alfrunes) [reply production/config/prod.yml.template:80]: I cherry-picked and amended your commit in #1365

**Comment by tranchitella** [comment]
Superseded by https://github.com/mendersoftware/integration/pull/1365

**Pull request closed without merging**